smallapp/unbound-control.c

```diff
@@ -372,6 +373,14 @@ static void print_extended(struct ub_stats_info* s)
 	PR_UL("rrset.cache.count", s->svr.rrset_cache_count);
 	PR_UL("infra.cache.count", s->svr.infra_cache_count);
 	PR_UL("key.cache.count", s->svr.key_cache_count);
+	/* applied RPZ actions */
+	for(i=0; i<UB_STATS_RPZ_ACTION_NUM; i++) {
+		if((enum rpz_action)s->svr.rpz_action[i] == RPZ_NO_OVERRIDE_ACTION)
+			continue;
```

Comment:
This must be wrong, the value is a statistics counter, it is never assigned to this variable and the enum is a valid value for the statistics counter.

Reply:
It is also wrong in remote.c, where the same line is copied from.